### PR TITLE
*: deploy main chain NeoFS contract during setup, fix #296

### LINKED ISSUE #296: Deploy mainchain neofs contract
## Is your feature request related to a problem? Please describe.
I'm always frustrated when I see chain dumps like devenv_mainchain.gz being used (CHAIN_URL).

## Describe the solution you'd like
Deploy this contract into the main chain during setup.

### PR DESCRIPTION
**Pull request by roman-khimov**
Not a lot of things to do here, but with this in place no dumps need to be created and you always have proper up-to-date NeoFS contract in dev-env, including development scenarios when you need to update it from some local copy.

Caveats:
 * waiter is a trivial sleep, sufficient here, see https://github.com/nspcc-dev/neo-go/issues/3244
 * .ir.env becomes dynamic, but it has to be present for docker-compose, see https://github.com/docker/compose/issues/3560
 * processing contract is not deployed deliberately, it's useless for notary-less
 * we will probably switch to notary-enabled when notary is no longer an extension

### CODE REVIEW

File: wallets/config.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (carpawell) [inline wallets/config.yml:1]: what this line is for?

File: Makefile [hunk 2]
```diff
@@ -119,6 +119,12 @@ up/bootstrap: get vendor/hosts
 		docker-compose -f services/$${svc}/docker-compose.yml up -d 2>&1 | tee -a docker-compose.err; \
 	done
 	@source ./bin/helper.sh
+	@docker exec main_chain neo-go wallet nep17 transfer --force --wallet-config /wallets/config.yml -r http://main-chain.neofs.devenv:30333 --from NfgHwwTi3wHAS8aFAN243C5vGbkYDpqLHP  --to NbUgTSFvPmsRxmGeWpuuGeJUoRoi6PErcM --token GAS --amount 1000
```
Comment (carpawell) [inline Makefile:122]: how temporary is it?

File: wallets/config.yml (hunk 1 above)
Comment (roman-khimov) [reply wallets/config.yml:1]: I have no idea. It's the same in `services/chain/config.yml`

File: Makefile (hunk 2 above)
Comment (roman-khimov) [reply Makefile:122]: What temporary?

Comment (carpawell) [reply Makefile:122]: i mean this looks like the most complicated target in our Makefiles. dont want to say it is bad but we tried to put such scripts in some `.sh` files before

Comment (roman-khimov) [reply Makefile:122]: 1. Things in `bin` are mostly reusable in some other contexts, deposit/tick/config are all relevant for manual execution, this one not so much (like `neofs-adm` below).
2. Those scripts are more generic and fancy, allowing overrides of various sorts, but here we have to use two different neo-go binaries and have to do very specific things.
3. `neofs-adm` invocations below are there, they're not scripted.
4. It's just four lines, really.

Comment (carpawell) [reply Makefile:122]: can accept it

**Review by carpawell — APPROVED** [review]

**Pull request merged**